test/ml/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(SUBSYS_NAME tests_ml)
+set(SUBSYS_DESC "Point cloud library ml module unit tests")
+PCL_SET_TEST_DEPENDENCIES(SUBSYS_DEPS ml)
+
+set(DEFAULT ON)
+set(build TRUE)
+PCL_SUBSYS_OPTION(build "${SUBSYS_NAME}" "${SUBSYS_DESC}" ${DEFAULT} "${REASON}")
+PCL_SUBSYS_DEPEND(build "${SUBSYS_NAME}" DEPS ${SUBSYS_DEPS} OPT_DEPS ${OPT_DEPS})
+
+if(NOT build)
+  return()
+endif()
+
+PCL_ADD_TEST(ml_kmeans test_ml_kmeans
```

Comment:
Please make it a single line and add an additional empty line to the of file (required in multiple files).

Reply:
Which one should I rewrite? 
```
PCL_ADD_TEST(ml_kmeans test_ml_kmeans FILES test_kmeans.cpp LINK_WITH pcl_gtest pcl_ml)
```
or
```
PCL_ADD_TEST(ml_kmeans test_ml_kmeans
            FILES test_kmeans.cpp LINK_WITH pcl_gtest pcl_ml)
```

Reply:
To the first - a single line.
Thats how they are in common at least.